tests/cnf/core/network/metallb/tests/frrk8-tests.go

```diff
@@ -479,11 +485,17 @@ var _ = Describe("FRR", Ordered, Label(tsparams.LabelFRRTestCases), ContinueOnFa
 
 				By("Creating FRR Hub pod on worker node 0")
```

Comment:
Just aksing, do we really need the ipforwarding enable  in all pod? I mean unless we have a multihop emulation there should be no forwarding, no?

Reply:
No, we don't. I only enable IP forwarding on hub pods that handle routing between two networks in the multihop scenario.